src/test/java/com/snowflake/kafka/connector/ConnectorConfigTest.java

```diff
@@ -622,4 +623,44 @@ public void testValidSchematizationForStreamingSnowpipe() {
     config.put(Utils.SF_ROLE, "ACCOUNTADMIN");
     Utils.validateConfig(config);
   }
+
+  @Test(expected = SnowflakeKafkaConnectorException.class)
```

Comment:
Can we put this in a new test file in /internal/streaming/SnowpipeStreamingFileTypeTest or something similar? since (i think?) this tests the SnowpipeStreamingFileType.VALIDATOR. if not, we should add additional tests for the validator

Reply:
IMO, all of these are all tested only through validate logic, so no need to put it in its native file.